### ISSUE TITLE
Table Sticky Header in IE

### ISSUE THREAD

**Issue by mbeckenbach** [angular/material2#12295]
#### Bug, feature request, or proposal:
feature request

#### What is the expected behavior?
Are there plans or hints on how to get sticky headers working for tables in IE?
Columns would be nice, but not needed.

#### What is the current behavior?
No sticky headers in IE...

#### What are the steps to reproduce?
Providing a StackBlitz reproduction is the *best* way to share your issue. <br/>
StackBlitz starter: https://goo.gl/wwnhMV<br/>

#### What is the use-case or motivation for changing an existing behavior?
IE is still the default browser for most enterprise customers. :-(

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular, Material: 6.3.1

**Comment by jrood** [comment]
I've also referenced rendering issues in Edge https://github.com/angular/material2/issues/11921

**Comment by ngoswald11404** [comment]
Is there any update on this?

**Comment by ajaymalhotra15** [comment]
Sticky header and footer is not working in IE Browser.

**Comment by tabruzzomufg** [comment]
any solution for this?

**Comment by mohankumarsivakumar** [comment]
yes , position : sticky is not supported in internet explorer 9 or above. so, the solution will be like moving this scrolling functionality to javascript so that it won't get affected based on the browser.
try opening the below index.html file in any browser and feel the solution

**Comment by JaxonWright** [comment]
Sticky footers are also still broken in IE11.

**Comment by patelatit53** [comment]
is there any update on this?

**Comment by crisbeto** [comment]
Closing since we don't support IE anymore.
